renderdoc/driver/gl/gl_hooks_egl.cpp

```diff
@@ -203,6 +203,13 @@ class EGLHook : LibraryHook, public GLPlatform
     GLWindowingData ret;
     EGLNativeWindowType window = 0;
 
+#if ENABLED(RDOC_ANDROID)
+    if(system == eWindowingSystem_Android)
+    {
+        window = (EGLNativeWindowType)data;
+    }
+    else
```

Comment:
I think it is possible to have it in this PR, just need to guard the XOpenDisplay call in the unknown case as the pbuffer creation depends on the window info only.